### DISCUSSION TITLE
How can you enable the table of contents (TOC) to always be expanded

### DISCUSSION THREAD

**Discussion by joshlk** [cotes2020/jekyll-theme-chirpy#1741] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

The default behaviour is for the table of contents (TOC) in posts to automatically expand and collapse as you scroll the page. Is there a way to make the TOC always be expanded (no collapsing)?

### Operations you have already tried

I have tried copying the [`_javascript/modules/components/toc.js`](https://github.com/cotes2020/jekyll-theme-chirpy/blob/v7.0.0/_javascript/modules/components/toc.js) file to my own repo and adding a `collapseDepth: 100` into the init parameters but that doesn't seem to help

**Comment by huanyushi** [comment, marked as answer]
Duplicate #1706

**Reply by joshlk** [reply]
Sorry, I missed this! I did do a search before posting 🤦
